### DISCUSSION TITLE
run introspection-engine.exe directly exit

### DISCUSSION THREAD

**Discussion by javacode123** [prisma/prisma#17985] [Q&A]
download_url: https://binaries.prisma.sh/all_commits/11e97a060a6eaa5ca0b29f3b612466b33fab1a21/windows/introspection-engine.exe.gz
os: win amd64
![image](https://user-images.githubusercontent.com/11720169/219885553-51ed3a3b-5295-4dc7-b599-f70f993f220e.png)

**Comment by nurul3101** [comment]
Hello @javacode123 👋 

Are you able to download the introspection engine?
Are you getting any error message?
Which Prisma command are you trying to invoke?

**Reply by javacode123** [reply]
i download introspect from 「https://binaries.prisma.sh/all_commits/11e97a060a6eaa5ca0b29f3b612466b33fab1a21/windows/introspection-engine.exe.gz」， and run ./introspection-engin.exe directly, also can see this「https://github.com/prisma/prisma/issues/17989」

**Reply by nurul3101** [reply]
It looks like your main use case is to use the [Prisma Go Client](https://github.com/prisma/prisma-client-go) which is now community maintained? (Based on this [comment](https://github.com/prisma/prisma/issues/17989#issuecomment-1436003311))

**Reply by javacode123** [reply]
yeap, i can use prisma go client. but i think the biggest problem is i can not run prisam on my laptop

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
